### DISCUSSION TITLE
JRuleNumberItem cannot be converted to String

### DISCUSSION THREAD

**Discussion by wollyd** [seaside1/jrule#181] [Q&A]
Hi,
In openHAB I have a Number Item (SlaapKamerS1_battery_level )
this is my jrule : 
```   
@JRuleWhenItemChange(item = SlaapKamerS1_battery_level )
    public void batteryAlarm(JRuleItemEvent event) {
```
When this is compiled I get the following error : 
> error: incompatible types: JRuleNumberItem cannot be converted to String

What am I missing here ?

Thanks in advance
Willy

**Comment by querdenker2k** [comment]
Your static import must be JRuleItemNames not JRuleItems. Annotation item
must be of type string.

Am 26. November 2023 11:57:06 schrieb wollyd ***@***.***>:

**Comment by wollyd** [comment]
Thanks a lot, that did the job.
